components/brave_shields/browser/https_everywhere_service.cc

```diff
@@ -105,14 +105,7 @@ HTTPSEverywhereService::HTTPSEverywhereService(
 }
 
 HTTPSEverywhereService::~HTTPSEverywhereService() {
-  Cleanup();
-}
-
-void HTTPSEverywhereService::Cleanup() {
-  GetTaskRunner()->PostTask(
-      FROM_HERE,
-      base::Bind(&HTTPSEverywhereService::CloseDatabase,
-                 AsWeakPtr()));
```

Comment:
`AsWeakPtr` would be empty when `Cleanup` is called from destructor, so this is noop